android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -50,25 +46,29 @@
 
 public class MapsFragment extends Fragment implements LocationListener, OnMapReadyCallback {
 
-    @BindView(R.id.autoCompleteTextView) AutoCompleteTextView searchView;
-
     private GoogleMap mMap;
     private Marker locationMarker;
     private List<String> searchItems = new ArrayList<>();
 
     private RealmDataRepository realmRepo = RealmDataRepository.getDefaultInstance();
     private List<Microlocation> mLocations = new ArrayList<>();
+    private android.support.v7.app.ActionBar mapToolbar;
```

Comment:
I named this toolbar earlier and then changed it to mapToolbar. I don't get what else to name :( Can you tell please?